src/main/java/org/galatea/starter/domain/IexHistoricalPrices.java

```diff
@@ -16,4 +15,41 @@
   private BigDecimal open;
   private long volume;
   private String date;
+
+  protected IexHistoricalPrices() {}
+
+  /**
+   * Constructor method for IexHistoricalPrices.
+   * @param symbol Stock symbol
+   * @param close Price at close
+   * @param high Adjusted high price for the day
+   * @param low Adjusted low price for the day
+   * @param open Adjusted open price for the day
+   * @param volume Volume of trades for the day
+   * @param date Date queried (formatted yyyy-mm-dd)
+   */
+  public IexHistoricalPrices(final String symbol, final BigDecimal close, final BigDecimal high,
+      final BigDecimal low, final BigDecimal open, final long volume, final String date) {
+    this.symbol = symbol;
+    this.close = close;
+    this.high = high;
+    this.low = low;
+    this.open = open;
+    this.volume = volume;
+    this.date = date;
+  }
```

Comment:
`@AllArgsConstructor` instead?

Reply:
Didn't know this was an available option - I'll use this annotation instead